containers/azure-functions-python-3/.devcontainer/Dockerfile

```diff
@@ -30,6 +29,9 @@ RUN \
     && apt-get autoremove -y \
     && apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
-        
+
+# Azure Functions Core Tools needs a place to save data
+ENV XDG_DATA_HOME=/usr/local/share
```

Comment:
Quick question - When connecting as the non-root vscode user, wouldn't this result in an access denied?

Reply:
Good catch. I've updated it to the a non-root user.